pandas/tests/series/methods/test_fillna.py

```diff
@@ -968,6 +967,6 @@ def test_fillna_parr(self):
         )
         expected = Series(arr)
 
-        filled = Series(parr.fillna(method="pad"))
+        filled = Series(ser.fillna(method="pad"))
```

Comment:
You can avoid Series(...) here, just do filled = ser.fillna(method="pad")

Reply:
🙈 🙈 this is why you shouldn't code late in the evening. Thanks, changed it ✅

Reply:
`Install dependencies` check inside `Python Dev / actions-311-dev (macOS-latest) ` fails though...

Reply:
This is unrelated, already opened an issue about it.

Reply:
No not really, something will have to investigate :) But it is not related to your pr, its failing on all prs